java/arcs/android/crdt/ParcelableRawEntity.kt

```diff
@@ -70,7 +70,8 @@ data class ParcelableRawEntity(
             @Suppress("GoodTime") // use Instant
             val expirationTimestamp = requireNotNull(parcel.readLong())
 
-            val rawEntity = RawEntity(id, singletons, collections, expirationTimestamp)
+            val rawEntity = RawEntity(id, singletons, collections)
+            rawEntity.expirationTimestamp = expirationTimestamp
```

Comment:
You could create a pseudo-constructor to RawEntity to allow for the inclusion of the timestamp there?

In RawEntity.kt:

```kotlin
data class RawEntity(.....) {...}

fun RawEntity(id: String, singletons: Map<...>, collections: Map<...>, expirationTimestamp: Long) = 
    RawEntity(id, singletons,  collections).also { it.expirationTimestamp =  expirationTimestamp }
```

Reply:
that's neat, thanks! done